app/Nova/Tools/TelescopeLink.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Nova\Tools;
+
+use Illuminate\View\View;
+use Laravel\Nova\Tool;
+
+class TelescopeLink extends Tool
```

Comment:
Psalm is finding an error here: 
```
ERROR: PropertyNotSetInConstructor - app/Nova/Tools/TelescopeLink.php:10:7 - Property Brewmap\Nova\Tools\TelescopeLink::$component is not defined in constructor of Brewmap\Nova\Tools\TelescopeLink and in any methods called in the constructor (see https://psalm.dev/074)
class TelescopeLink extends Tool
```

You can add `<directory name="app/Nova/Tools"/>` in `psalm.xml` in section `<PropertyNotSetInConstructor>` to supress it (as we cannot do anything with that).